### ISSUE TITLE
Block List, Content from settings not included when using copy/paste

### ISSUE THREAD

**Issue by limpa123** [umbraco/Umbraco-CMS#8905]
When copy/pasting a block list item the values from the Settings model is not included.

**Comment by nathanwoulfe** [comment]
Thanks @limpa123, I've verified this on a clean 8.7 install:

- created a block list
- created a block element
- created a settings element for the block
- added a block, added settings
- copied the block
- pasted from clipboard
- no settings. Sad. 

This appears to be by design - the `copyBlock` method looks like the below, where `block.content` is the data added to the clipboard:

```javascript
    function copyBlock(block) {
        clipboardService.copy(clipboardService.TYPES.ELEMENT_TYPE, block.content.contentTypeAlias, block.content, block.label);
    }
```

It would be possible (not tested, just me spitballing ideas) to pass a different object in place of the `block.content` parameter, instead setting something like `{ content: block.content, settings: block.settings }`. I haven't looked at how the block is pasted, but  at some point the `block.content` data must get pushed back onto the block array, so should be able to add the settings in the same way.

Pinging @nielslyngsoe here too, in case there are reasons for not copying settings in the current implementation. Haven't labelled as a bug/feature for the same reason.

**Comment by nielslyngsoe** [comment]
Hi @limpa123, thanks for highlighting this, as it also highlights that you as a user would like this feature.

It's not part of the original feature, as we currently just have implemented copying of types "ElementType" and "Raw" data. And therefore we just copy the content part of a block.

But never the less what @nathanwoulfe  describes is somehow the approach that needs to be taken, it does though require a little more than just that. Basically, we need to invent a new type to be stored in the clipboard, this type should also include the layout object (so we can copy layout properties, there are none for BlockList but other future editors could have this).

**Comment by nielslyngsoe** [comment]
I'm closing this one in favor of the feature request. https://github.com/umbraco/Umbraco-CMS/issues/8990

**Comment by warrenbuckley** [comment]
Is fixed in this PR and due for 8.10 release
https://github.com/umbraco/Umbraco-CMS/pull/9258
